### PR TITLE
Improvement disable control button when change state we do not redraw panel

### PR DESCRIPTION
**Pull request by vvlladd28**
When studying the code, it was found that the optimal movement, which causes a complete redrawing of the control panel, when changes disable state.

This PR makes the following changes: 

- When we change disable state, we change CSS class and internal state
- When you lock a button, we forcibly render it inactive
- Changed CSS styles to remove class duplication for different DOM element

### CODE REVIEW

**Comment by vvlladd28** [comment]
Please review my PR @Falke-Design

File: cypress/integration/toolbar.spec.js [hunk 1]
```diff
@@ -408,7 +408,28 @@ describe('Testing the Toolbar', () => {
...
+
+    cy.window().then(({ map }) => {
+      map.pm.Toolbar.setButtonDisabled('drawPolygon', true);
+    });
+
+    cy.window().then(() => {
+      expect(eventFired).to.equal('drawPolygon');
+      cy.toolbarButton('polygon')
+        .closest('.button-container')
+        .should('have.not.class', 'active');
```
Comment (Falke-Design) [inline cypress/integration/toolbar.spec.js:432]: Move this block above
```
 cy.window().then(({ map }) => {
      map.pm.Toolbar.setButtonDisabled('drawPolygon', true);
    });
```
And then add a check if the event / click happens after disableing the button

**Review by Falke-Design — CHANGES_REQUESTED** [review]
Why do you think it is important to merge this? Clearly the difference is that the Toolbar is not rendered new, but every change on the Toolbar cause a re-render so why should we handle this different?

Anyway thank you for your contribution.

**Comment by vvlladd28** [comment]
For example, I can give an example of [code](https://github.com/Leaflet/Leaflet/blob/4f639a85efffa49c3e64a07dc0b6f5aa73f13449/src/control/Control.Zoom.js#L102) from the leaflet library, they have buttons for zooming the map and don't render after disable. The redraw render is complex for the browser.

File: cypress/integration/toolbar.spec.js (hunk 1 above)
Comment (vvlladd28) [reply cypress/integration/toolbar.spec.js:434]: Thank you, fixed.

**Review by codeofsumit — APPROVED** [review]

**Pull request merged**